app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
```

Comment:
- [ ] The views in devise for the registration and login have to be modified to match the wireframes and their styling. Kindly do so.

Your design
![image](https://github.com/Kwaku28/foodbliss_blog/assets/81932478/c863eb6b-3fae-4746-b716-71d8bf974c68)

Given wireframe
![image](https://github.com/Kwaku28/foodbliss_blog/assets/81932478/1ce704cc-a4c9-4c83-97f6-5e14c825363b)